numbers/list.py

```diff
@@ -0,0 +1,26 @@
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), "../.env")
+load_dotenv(dotenv_path)
+
+NEXMO_API_KEY = os.getenv("NEXMO_API_KEY")
+NEXMO_API_SECRET = os.getenv("NEXMO_API_SECRET")
+NUMBER_SEARCH_CRITERIA = os.getenv("NUMBER_SEARCH_CRITERIA")
+NUMBER_SEARCH_PATTERN = os.getenv("NUMBER_SEARCH_PATTERN")
+
+import nexmo
+
+client = nexmo.Client(key=NEXMO_API_KEY, secret=NEXMO_API_SECRET)
+
+responseData = client.get_account_numbers(
+    {"pattern": NUMBER_SEARCH_CRITERIA, "search_pattern": NUMBER_SEARCH_PATTERN}
+)
+
+print(
+    f'Here are {str(len(responseData["numbers"]))} of the {str(responseData["count"])} matching numbers in your account:'
```

Comment:
You don't need the `str` wrapped around these values. f-strings will automatically format numbers etc into strings for you.

Reply:
f-strings for the win, though!